src/source/reporting.rst

```diff
@@ -206,6 +206,8 @@ The only valid ``operator`` at the moment is the set operator ``in``, and requir
     ]
 
 
+.. note:: Currently the API **does not allow to filter on the current status of ads**, which means the reporting numbers apply for **all** ads, including those that have been deleted in the meantime. Depending on API clients feedback, we might extend support for such filtering in the future.
```

Comment:
you say "Currently.." but I don't think we have any intention of changing that?

Reply:
no indeed, I'll rephrase it a bit. 
Who knows in the distant future we may be able to pull it off if there is a demand for it